tools/cldr-code/src/main/java/org/unicode/cldr/tool/ShowLocaleCoverage.java

```diff
@@ -601,18 +568,18 @@ public static void showCoverage(Anchors anchors, Matcher matcher, Set<String> lo
                     languageToRegion.put(ltc.transform(ltp.getLanguageScript()), country);
                 }
             }
+            languageToRegion = ImmutableMultimap.copyOf(languageToRegion);
 
             fixCommonLocales();
 
-            System.out.println(Joiner.on("\n").join(languageToRegion.keyValuesSet()));
+            System.out.println(Joiner.on("\n").join(languageToRegion.asMap().entrySet()));
```

Comment:
i see this is an *output* file. Are you on a a platform that expects `\r\n`? 

```suggestion
            System.out.println(Joiner.on(System.lineSeparator()).join(languageToRegion.asMap().entrySet()));
```

Reply:
No, it expects "\n"; I think this is unrelated.